### PR TITLE
[BrokenLinksH2] Corrected Azure pricing link

### PR DESCRIPTION
**Pull request by kturetz**
Fixed broken link on Line 23.

### CODE REVIEW

File: microsoft-365/compliance/double-key-encryption.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ ms.custom: admindeeplinkCOMPLIANCE
 
 [!include[Purview banner](../includes/purview-rebrand-banner.md)]
 
-> *Applies to: Microsoft Purview Double Key Encryption, [Microsoft Purview](https://www.microsoft.com/microsoft-365/business/compliance-management), [Azure Information Protection](https://azure.microsoft.com/pricing/details/information-protection)*
+> *Applies to: Microsoft Purview Double Key Encryption, [Microsoft Purview](https://www.microsoft.com/microsoft-365/business/compliance-management), [Azure Information Protection](https://azure.microsoft.com/en-us/pricing/)*
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/double-key-encryption.md]:
```suggestion
> *Applies to: Microsoft Purview Double Key Encryption, [Microsoft Purview](https://www.microsoft.com/microsoft-365/business/compliance-management), [Azure Information Protection](https://azure.microsoft.com/pricing/)*
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @kturetz, kindly commit the suggested change to remove the locale from the link. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by kturetz** [comment]
**Global effort to fix broken links** 

@krowley

The Content & Learning team is fixing broken links on docs.microsoft.com for the rest of H2. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only link fixes and does not change other content. Please review within five business days and merge, or comment in the PR with any changes you'd like to see. Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**